modelci/hub/publish.py

```diff
@@ -15,22 +15,31 @@
 import os
```

Comment:
I don't like this name. `publish` is a verb, better use noun. How about `registrar` @HuaizhengZhang

Reply:
Should we add a TODO here? Make this change in this PR will make it too complicated.

Reply:
you may change to `modelci/hub/registrar.py` rather than `modelci/hub/register/publish.py`

Reply:
We use the expression `publish` in CLI and front end to represent a process of registering a model to the hub. Directly renaming the `publish.py` will make CLI and front end a bit confusing.  So I recommend to keep this name. Moving it to a package `registrar` would be a better choice for both renaming the process and further expanding for this module. By the way, since converter, which is of same level as publishing process,  is wrapped in a package module. I think publish deserves a package as well.